baseplate/thing_id.py

```diff
@@ -0,0 +1,42 @@
+"""Thing ID prefix and validation."""
+
+
+class ThingPrefix:
+    """Thing object prefixes."""
+
+    COMMENT = "t1_"
+    ACCOUNT = "t2_"
+    LINK = "t3_"
+    MESSAGE = "t4_"
+    SUBREDDIT = "t5_"
+    AWARD = "t6_"
```

Comment:
@spladug Do we want to use Enum's for this?

```
class ThingPrefix(enum.Enum):
    """Thing object prefixes."""
    comment = "t1_"
    account = "t2_"
    link = "t3_"
    message = "t4_"
    subreddit = "t5_"
    award = "t6_"
```

You have to say `ThingPrefix.comment.value` to get the actual string value so I'm not sure, here are things we would have to do to work with it:

```
# Get the actual prefix from enum
ThingPrefix.comment.value

# Get the enum by value
ThingPrefix("t2_")
```